client/src/components/CategoryNav.tsx

```diff
@@ -1,23 +1,31 @@
-import { useState } from "react"
-import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs"
+import { useState } from "react";
+import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs";
 
+interface CategoryNavProps {
 interface CategoryNavProps {
   onCategoryChange: (category: string) => void;
   categories: string[];
+  categories: string[];
 }
 
```

Comment:
Looks like a merge conflict went sideways here
```suggestion

interface CategoryNavProps {
  onCategoryChange: (category: string) => void;
  categories: string[];
 }

```